_static/css/aplusmanual.css

```diff
@@ -0,0 +1,14 @@
+@import url("../default.css");
+
+/* Render box to show the HTML output for the rst syntax */
+.html-box {
```

Comment:
What is this really? Where is it used?

Reply:
I am using the .html-box class to present some content, but I will probably remove that class. 
If you are talking about the `aplusmanual.css` file, this is a temporary file to apply some personalised styles, but in the end, I will add those changes to the `course.css` file